contentcuration/contentcuration/frontend/shared/data/resources.js

```diff
@@ -1196,6 +1196,15 @@ export const Invitation = new Resource({
   tableName: TABLE_NAMES.INVITATION,
   urlName: 'invitation',
   indexFields: ['channel'],
+
+  accept(id) {
+    const changes = { accepted: true };
+    return client.patch(window.Urls.invitationDetail(id), changes).then(() => {
```

Comment:
@jayoshih This seems good for now, but for the future, a "proper" fix for the failing client mock issue is to move that `accept` functionality out of `resources.js` and into a vuex action.  I can follow up on that after launch.

Reply:
I would prefer that we keep it here to keep all the api calls in one place rather than move it due to tests